packages/blocks/src/api/constants.js

```diff
@@ -46,6 +41,10 @@ export const __EXPERIMENTAL_STYLE_PROPERTY = {
 		value: [ 'color', 'text' ],
 		support: [ 'color' ],
 	},
+	linkColor: {
```

Comment:
How about we call this `colorLink` so it's alphabetized and consecutive to `color`?

Reply:
Nah, forget about this. We also have `backgroundColor`, etc. Too difficult to change the naming schema at this point.